evolver/base.py

```diff
@@ -2,6 +2,33 @@
 import logging
 
 import pydantic
+import pydantic_core
+
+
+def require_all_fields(cls):
+    """ Decorate a model mutating it to one where all fields are required.
+
+        Example:
+            ```python
+            import pydantic
+            from evolver.base import require_all_fields
+
+            class MyModel(pydantic.BaseModel):
+                a: int = 3
+
+            @require_all_fields
+            class MyModelWithoutDefaults(MyModel):
+                ...
+            ```
+    """
+    for field in cls.model_fields:
```

Comment:
I looked to see if there was a more pydantic/fastapi way of doing this, but couldn't find anything canonical so came up with this. If there is a more correct way of doing this within the framework, I'd rather use that than this.

Another option is to create a child class of ``FieldInfo`` with ``is_required`` overridden to always return ``True``, however, I thought it simple to mutate the attrs on the single model class than create new class instances for each ``Field``.